### ISSUE TITLE
Class blade.compiler does not exist

### ISSUE THREAD

**Issue by oriceon** [yajra/laravel-datatables#2008]
Hi there. I'm trying to migrate to Laravel 5.8 from 5.7, and I've also switched to the new version of datatable 9 but after upgrading I get an error:

```Class blade.compiler does not exist```

And i see you using that in ```C:\SlotSys\www\aplicatie\vendor\yajra\laravel-datatables-oracle\src\Utilities\Helper.php``` at compileBlade method.

Any ideea what happened?

Thanks.

**Comment by yajra** [comment]
Maybe some cache issues? Just updated an app today and all seems to work fine.

Are you still having this issue?

**Comment by ultrono** [comment]
Just upgraded to Laravel 5.8.8 and got the same error. `vendor\yajra\laravel-datatables-oracle\src\Utilities\Helper.php` was the first file I found that mentioned the issue, but wasn't the cause.

I my case, I tracked the error down to a custome blade service provider, where I registered some unused blade directives, from within the service providers `boot` method. Removing them resolved the issue and allowed the `@php artisan package:discover --ansi` command to complete.
